lib/griddler/mandrill/adapter.rb

```diff
@@ -63,12 +64,20 @@ def full_email(contact_info)
       end
 
       def attachment_files(event)
-        attachments = event[:attachments] || Array.new
-        attachments.map do |key, attachment|
+        files(event, :attachments)
+      end
+
+      def inline_images(event)
+        files(event, :images)
+      end
+
+      def files(event, key)
+        files = event[key] || Array.new
```

Comment:
I prefer `wrap` as that is [used elsewhere already](https://github.com/wingrunr21/griddler-mandrill/blob/master/lib/griddler/mandrill/adapter.rb#L44).